apps/api/src/app/channeling/entities/channeling_session.entity.ts

```diff
@@ -0,0 +1,10 @@
+import { Schema } from 'mongoose';
+
+export const ChannelingSessionEntity = new Schema({
+  channelingSessionId: String,
```

Comment:
_id field will be assigned by mongoose automatically

Reply:
removed _id